benchmarks/operator_benchmark/pt/channel_shuffle_test.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import operator_benchmark as op_bench
+import torch
+
+
+"""Microbenchmarks for channel_shuffle operator."""
+
+
+# Configs for PT channel_shuffle operator
+channel_shuffle_long_configs = op_bench.cross_product_configs(
```

Comment:
Could you try to reduce the number of configs here? This config will create 5 x 4 x 4 x 4 x 4 x 2 = 2560 different test cases. Ideally we would like each operator to have less than 100 different tests.

Reply:
Thanks @mingzhe09088. Tai, can you reduce each param to only two values?